data-manipulation/hashing/ripemd160/hash-data-using-ripemd160.yml

```diff
@@ -0,0 +1,27 @@
+rule:
+  meta:
+    name: hash data using RIPEMD160
+    namespace: data-manipulation/hashing/ripemd160
+    authors:
+      - raymond.leong@mandiant.com
+    scope: function
+    references:
+      - https://en.wikipedia.org/wiki/RIPEMD-160
+    examples:
+      - 35F9CFE5110471A82E330D904C97466A:0x10009340
+  features:
+    - and:
+      - or:
+        - and:
+          - description: Magic initialization constants used in RIPEMD-160
+          - number: 0x67452301 = also used in MD5, SHA1, RIPEMD-128, RIPEMD-256, and RIPEMD-320
```

Comment:
I like that these inline descriptions can point analysts to the alternative algorithms!